tools/builder/react.js

```diff
@@ -11,9 +10,7 @@ import PluginError from 'plugin-error';
 import rename from 'gulp-rename';
 import saveLicense from 'uglify-save-license';
 import sourcemaps from 'gulp-sourcemaps';
-import tap from 'gulp-tap';
 import webpack from 'webpack';
-import { JSDOM } from 'jsdom';
```

Comment:
Do we still need `jsdom` in the repository at all, after it's removed from here?

Reply:
Hehe, I was also hoping to get rid of it :smile: Unfortunately, we're still using `jsdom-global` [in our tests](https://github.com/Automattic/jetpack/blob/4e716c7bdf6aae6dd5fe39c9029a509c7c2bee6f/tests/runner.js#L39), which in turn [depends on `jsdom`](https://github.com/rstacruz/jsdom-global#install). So I'm afraid we need to keep it for a little longer :sweat_smile: